.github/workflows/check-artifact-changes.yml

```diff
@@ -0,0 +1,42 @@
+name: Check Artifact Changes
+
+on:
+  pull_request:
+    types: [ opened, reopened, labeled, unlabeled, synchronize ]
+    paths-ignore: [ '.changes/**', '.github/**', 'tests/**', '**.md', '**.yml' ]
+
+  workflow_dispatch:
+
+jobs:
+  check-artifact-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check for changes in core/dbt/artifacts
+        id: check_changes
+        run: |
+          # Check for changes in dbt/artifacts
+          CHANGED_FILES=$(git diff --name-only origin/${GITHUB_BASE_REF} origin/${GITHUB_HEAD_REF} | grep 'core/dbt/artifacts/')
```

Comment:
Could we exclude matching `core/dbt/artifacts/README.md` as part of this search?

Reply:
We're ignoring all `*.md` files, so we should be fine:

https://github.com/dbt-labs/dbt-core/pull/9553/files#diff-a7bcbf79c3b395a73ebf1356d7c6a129c82f2f01d385cfbf82cfc4af21f5b3c0R6